client/reducers/foods.js

```diff
@@ -1,10 +1,19 @@
 import { GET_FOODS_SUCCESS } from './../actions/foods'
+import { GET_CATEGORIES_SUCCESS } from './../actions/categories'
 
 export default function foodReducer (state = [], action) {
   switch (action.type) {
     case GET_FOODS_SUCCESS:
       return action.foods
+    default:
+      return state
+  }
+}
 
+export default function categoryReducer (state = {}, action) {
```

Comment:
Please write a test for this reducer too in `tests/client/reducer/foods.test.js`